clients/extension/src/navigation/views.tsx

```
@@ -16,92 +15,83 @@
-  uploadQr: <ImportQRPage />,
-  vaults: (
+  uploadQr: () => <ImportQRPage />,
+  vaults: () => (
     <ActiveVaultGuard>
       <VaultsPage />
     </ActiveVaultGuard>
   ),
-  deposit: (
+  deposit: () => (
     <ActiveVaultGuard>
       <>{/* <>TODO: Implement Deposit page</> */}</>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)